src/App.tsx

```diff
@@ -1,61 +1,162 @@
+import React, { useState } from 'react';
+
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { Todo } from './type/Todo';
+import { User } from './type/User';
+
+import { TodoList } from './components/TodoList';
+
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+function getUser(userId: number): User | null {
+  const findUser = usersFromServer.find(
+    user => user.id === userId,
+  );
+
+  return findUser || null;
+}
+
+const todos: Todo[] = todosFromServer.map(
+  todo => ({
+    ...todo,
+    user: getUser(todo.userId) || null,
+  }),
+);
+
+export const App: React.FC = () => {
+  const [text, setText] = useState('');
+  const [targetUser, setTargetUser] = useState('');
+  const [correctList, setCorrectList] = useState([...todos]);
+  const [errorText, setErrorText] = useState(true);
```

Comment:
[useful link](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig)

```suggestion
  const [isErrorText, setIsErrorText] = useState(true);
```